src/EnrollActivateEmailController.js

```diff
@@ -46,7 +46,7 @@ define([
       title: _.partial(Okta.loc, 'email.enroll.title', 'login'),
       noButtonBar: false,
       autoSave: true,
-      save: _.partial(Okta.loc, 'oform.verify', 'login'),
+      save: _.partial(Okta.loc, 'oform.verify', 'login'), // TODO: deprecated by mfa.challenge.verify
```

Comment:
what is stopping us from replacing `oform.verify` with `mfa.challenge.verify` in this PR ?

Reply:
In theory, if customer overrides `oform.verify` but not `mfa.challenge.verify` will be broken.
But sounds like very very rare?

Reply:
yeah, I think that is rare.

Reply:
I'll make change to master in separate PR.